src/EditorFeatures/Text/Shared/Extensions/ITextSnapshotLineExtensions.cs

```diff
@@ -67,13 +67,18 @@ internal static class ITextSnapshotLineExtensions
         /// <summary>
         /// Determines whether the specified line is empty or contains whitespace only.
         /// </summary>
-        public static bool IsEmptyOrWhitespace(this ITextSnapshotLine line)
+        public static bool IsEmptyOrWhitespace(this ITextSnapshotLine line, int startIndex = 0, int endIndex = -1)
```

Comment:
Maybe add an argument check to validate that `startIndex` is not less than zero?

Reply:
➡️ Check added in 42baa98e7fb23c6820fa7dccb247b0fedb7c6374